src/FunctionApps/python/IntakePipeline/transform.py

```diff
@@ -13,52 +13,108 @@ def find_patient_resources(bundle: dict) -> dict:
     """Standardize name and phone, geocode the address"""
 
     for resource in find_patient_resources(bundle):
         patient = resource.get("resource")
+        if "extension" not in patient:
+            patient["extension"] = []
 
         # Transform names
         for name in patient.get("name", []):
-            process_name(name)
+            process_name(name, patient)
 
         # Transform phone numbers
+        raw_phones = []
+        std_phones = []
         for telecom in patient.get("telecom", []):
             if telecom.get("system") == "phone" and "value" in telecom:
-                telecom["value"] = transform_phone(telecom["value"])
+                transformed_phone = transform_phone(telecom["value"])
+                raw_phones.append(telecom["value"])
+                std_phones.append(transformed_phone)
+                telecom["value"] = transformed_phone
+        any_diffs = any(
+            [raw_phones[i] != std_phones[i] for i in range(len(raw_phones))]
+        )
+        patient["extension"].append(
+            {
+                "url": "http://usds.gov/fhir/phdi/StructureDefinition/phone-was-standardized",  # noqa
+                "valueBoolean": any_diffs,
+            }
+        )
 
+        raw_addresses = []
+        std_addresses = []
         for address in patient.get("address", []):
             # Generate a one-line address to pass to the geocoder
             one_line = " ".join(address.get("line", []))
             one_line += f" {address.get('city')}, {address.get('state')}"
             if "postalCode" in address and address["postalCode"]:
                 one_line += f" {address['postalCode']}"
+            raw_addresses.append(one_line)
 
             geocoded = geocode(client, one_line)
+            std_one_line = ""
             if geocoded:
                 address["line"] = geocoded.address
                 address["city"] = geocoded.city
                 address["state"] = geocoded.state
                 address["postalCode"] = geocoded.zipcode
+                std_one_line = f"{geocoded.address} {geocoded.city}, {geocoded.state} {geocoded.zipcode}"  # noqa
+                std_addresses.append(std_one_line)
 
                 if "extension" not in address:
                     address["extension"] = []
 
                 address["extension"].append(
                     {
-                        "url": "http://hl7.org/fhir/StructureDefinition/geolocation",
+                        "url": "http://hl7.org/fhir/StructureDefinition/geolocation",  # noqa
                         "extension": [
                             {"url": "latitude", "valueDecimal": geocoded.lat},
                             {"url": "longitude", "valueDecimal": geocoded.lng},
                         ],
                     }
                 )
+        any_dffs = any(
+            [raw_addresses[i] != std_addresses[i] for i in range(len(raw_addresses))]
```

Comment:
In theory, if raw_addresses has length n, std_addresses has length n+1, and the values in the first n are the same, this could return a false positive.  Maybe consider this?
```
any_dffs = (len(raw_addresses) != len(std_addresses)) or any(
            [raw_addresses[i] != std_addresses[i] for i in range(len(raw_addresses))]
```